webapp/crashstats/crashstats/tests/test_views.py

```diff
@@ -72,6 +77,36 @@
 }
 
 
+def build_crash_data():
+    crash_id = create_new_ooid()
+
+    raw_crash = copy.deepcopy(_SAMPLE_META)
+    processed_crash = copy.deepcopy(_SAMPLE_PROCESSED)
+    processed_crash["uuid"] = crash_id
+    processed_crash["date_processed"] = date_to_string(date_from_ooid(crash_id))
+    return crash_id, raw_crash, processed_crash
+
+
+def upload_crash_data(s3_helper, raw_crash, processed_crash):
+    """Validate crash data and upload it to crash bucket"""
+    crash_id = processed_crash["uuid"]
+    bucket = s3_helper.get_crashstorage_bucket()
+
+    validate_instance(raw_crash, RAW_CRASH_SCHEMA)
```

Comment:
This guarantees the data we're building is valid raw crash data. This is pretty important when testing the webapp.